sdk/include/wdf/kmdf/1.9/wdf.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * wdf.h
+ *
+ * Windows Driver Framework - Primary Header File
+ *
+ * This file is part of the ReactOS wdf package.
+ *
+ * Contributors:
+ *   Created by Benjamin Aerni <admin@bennottelling.com>
+ *
+ * Intended Usecase:
+ *   Kernel mode drivers
+ *
+ * THIS SOFTWARE IS NOT COPYRIGHTED
+ *
+ * This source code is offered for use in the public domain. You may
+ * use, modify or distribute it freely.
+ *
+ * This code is distributed in the hope that it will be useful but
+ * WITHOUT ANY WARRANTY. ALL WARRANTIES, EXPRESS OR IMPLIED ARE HEREBY
+ * DISCLAIMED. This includes but is not limited to warranties of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ */
+#ifndef _WDF_H_
+#define _WDF_H_
+
+
+typedef VOID (*WDFFUNC) (VOID);
```

Comment:
You'll need to add `NTAPI` to pretty much any function declaration or function pointer type. MS's WDK headers assume that you're compiling with `/GR`, which makes it the default. GCC no longer has the corresponding option, so all our headers have to be explicit about this.